tests/test_job.py

```diff
@@ -943,7 +972,82 @@ def test_dependents_are_met_if_dependency_is_deleted(self):
 
         assert dependent_job.dependencies_are_met()
         assert dependent_job.get_status() == JobStatus.QUEUED
-    
+
+    def test_rpush_fixture(self):
+        fixtures.rpush('foo', 'bar')
+        assert as_text(self.testconn.lrange('foo', 0, 0)[0]) == 'bar'
```

Comment:
Can we do `self.testconn.lrange('foo', 0, 0).decode()[0] == 'bar'` here? I want to eventually get rid of `as_text` because this was used for Python 2/3 compatibility purposes.

Reply:
Done.